Lib/sqlite3/__main__.py

```diff
@@ -35,57 +39,65 @@ def runpy(self, source, filename="<input>", symbol="single"):
 
     def printhelp(self, ignored):
         print("Enter SQL code and press enter.")
+        return None
 
     def runsource(self, source, filename="<input>", symbol="single"):
         keywords = {
-            "version": lambda x: print(f"{sqlite3.sqlite_version}"),
-            "help": self.printhelp,
-            "quit()": self.runpy,
-            "quit": self.runpy,
+            ".version": lambda x: print(f"{sqlite3.sqlite_version}"),
+            ".help": self.printhelp,
+            ".quit": lambda x: sys.exit(0),
```

Comment:
None of actions use the argument. Maybe remove it?
```python
if source in keywords:
    keywords[source]()
else:
    self.runsql(source)
```
Or maybe even use a pattern matching?

Reply:
Yeah, I need a good excuse to use pattern matching 😆